website/docs/guides/policies.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "vault"
+page_title: "Vault Policies and the Terraform Vault Provider"
+sidebar_current: "docs-vault-policies-and-tfvp"
+description: |-
+  Vault Policies and the Terraform Vault Provider
+
+---
+
+# Vault Policies and the Terraform Vault Provider
+
+Vault uses policies to govern the behavior of clients and instrument Role-Based
+Access Control (RBAC) by specifying access privileges (authorization).
+
+Vault's [root](https://developer.hashicorp.com/vault/docs/concepts/policies#root-policy)
+policy is capable of performing every operation for all paths. Root tokens are
+tokens that have the `root` policy attached to them. Root tokens are useful in
+development but should be carefully guarded in production.
+
+In environments where permissions are least privilege, the Terraform Vault
+Provider should not be given a token that has the root policy assigned to it.
+Instead, the Vault provider should be given a token that limits its actions to only
+the operations that it needs to provision Vault's resources.
+
+For example, the following policy would limit the Vault provider to enabling and
```

Comment:
nit: wondering if it is worth highlighting full lifecycle management
```suggestion
For example, the following policy would limit the Vault provider to managing the lifecycle for the
```

Reply:
Done in https://github.com/hashicorp/terraform-provider-vault/pull/2194/commits/ec4bb85f1b144efcbb5b45def84d4719dd140c65